src/app/search/state/search.service.spec.ts

```diff
@@ -49,4 +51,14 @@ describe('SearchService', () => {
   it('should not crash on null advancedSearchObject', inject([SearchService], (service: SearchService) => {
     expect(service.hasSearchText(null, null, null)).toEqual(false);
   }));
+
+  it('should create image provider', inject([SearchService], (service: SearchService) => {
+    const filtered: [Array<any>, Array<any>] = service.filterEntry(elasticSearchResponse, 201);
+    const tools = filtered[0];
+    expect(filtered[1].length).toBe(0);
```

Comment:
In conjunction with my other comment, should test that the workflow shows up here, and perhaps make sure it doesn't have imgProvider and imgProviderUrl

Reply:
Included a test for the workflows to insure that it does not have imageProvider or imageProviderURL